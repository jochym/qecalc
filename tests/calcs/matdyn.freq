 &plot nbnd=   9, nks= 151 /
            0.000000  0.577350  0.000000
  253.7291  260.9608  326.2879  423.7423  459.6642  487.2487
  717.8538  723.2940  758.5226
            0.000000  0.565803  0.000000
  253.6604  260.8558  326.2134  422.6472  459.6308  488.7963
  717.0286  723.3681  759.1836
            0.000000  0.554256  0.000000
  253.4540  260.5368  325.9894  419.6720  459.5253  493.1198
  714.6872  723.5899  761.0236
            0.000000  0.542709  0.000000
  253.1092  259.9922  325.6141  415.4838  459.3320  499.5250
  711.1256  723.9581  763.7187
            0.000000  0.531162  0.000000
  252.6248  259.2040  325.0849  410.7097  459.0256  507.3405
  706.6324  724.4694  766.9349
            0.000000  0.519615  0.000000
  251.9990  258.1488  324.3981  405.7908  458.5731  516.0653
  701.4135  725.1174  770.4046
            0.000000  0.508068  0.000000
  251.2298  256.8006  323.5494  401.0031  457.9354  525.3492
  695.5957  725.8908  773.9244
            0.000000  0.496521  0.000000
  250.3144  255.1322  322.5342  396.5110  457.0708  534.9420
  689.2518  726.7712  777.3320
            0.000000  0.484974  0.000000
  249.2499  253.1182  321.3477  392.4076  455.9376  544.6560
  682.4225  727.7312  780.4872
            0.000000  0.473427  0.000000
  248.0327  250.7368  319.9849  388.7394  454.4980  554.3434
  675.1307  728.7320  783.2601
            0.000000  0.461880  0.000000
  246.6588  247.9713  318.4413  385.5222  452.7214  563.8843
  667.3907  729.7222  785.5247
            0.000000  0.450333  0.000000
  244.8112  245.1239  316.7127  382.7513  450.5872  573.1799
  659.2139  730.6369  787.1571
            0.000000  0.438786  0.000000
  241.2530  243.4233  314.7953  380.4081  448.0868  582.1491
  650.6116  731.3980  788.0357
            0.000000  0.427239  0.000000
  237.2997  241.5518  312.6862  378.4656  445.2244  590.7264
  641.5970  731.9159  788.0425
            0.000000  0.415692  0.000000
  232.9607  239.5044  310.3830  376.8923  442.0170  598.8610
  632.1858  732.0925  787.0665
            0.000000  0.404145  0.000000
  228.2511  237.2758  307.8839  375.6552  438.4929  606.5160
  622.3965  731.8244  785.0063
            0.000000  0.392598  0.000000
  223.1913  234.8610  305.1878  374.7221  434.6895  612.2502
  613.6674  731.0077  781.7737
            0.000000  0.381051  0.000000
  217.8061  232.2549  302.2941  374.0630  430.6507  601.7698
  620.3032  729.5429  777.2976
            0.000000  0.369504  0.000000
  212.1248  229.4529  299.2026  373.6506  426.4238  590.9803
  626.4228  727.3395  771.5271
            0.000000  0.357957  0.000000
  206.1799  226.4506  295.9131  373.4611  422.0568  579.9076
  632.0356  724.3209  764.4351
            0.000000  0.346410  0.000000
  200.0072  223.2438  292.4251  373.4736  417.5960  568.5786
  637.1596  720.4289  756.0211
            0.000000  0.334863  0.000000
  193.6446  219.8281  288.7376  373.6696  413.0835  557.0211
  641.8201  715.6267  746.3140
            0.000000  0.323316  0.000000
  187.1316  216.1988  284.8481  374.0330  408.5568  545.2635
  646.0480  709.9022  735.3742
            0.000000  0.311769  0.000000
  180.5079  212.3503  280.7522  374.5490  404.0471  533.3360
  649.8784  703.2701  723.2948
            0.000000  0.300222  0.000000
  173.8126  208.2757  276.4425  375.2038  399.5804  521.2710
  653.3490  695.7727  710.2033
            0.000000  0.288675  0.000000
  167.0826  203.9668  271.9080  375.9840  395.1774  509.1043
  656.4984  687.4805  696.2604
            0.000000  0.277128  0.000000
  160.3517  199.4131  267.1321  376.8759  390.8547  496.8771
  659.3649  678.4915  681.6597
            0.000000  0.265581  0.000000
  153.6489  194.6023  262.0922  377.8657  386.6263  484.6381
  661.9851  666.6247  668.9291
            0.000000  0.254034  0.000000
  146.9977  189.5203  256.7578  378.9386  382.5047  472.4451
  651.4049  658.9396  664.3930
            0.000000  0.242487  0.000000
  140.4149  184.1518  251.0898  378.5024  380.0791  460.3674
  636.2698  648.6880  666.6191
            0.000000  0.230940  0.000000
  133.9104  178.4805  245.0395  374.6321  381.2713  448.4872
  621.5016  638.3535  668.6900
            0.000000  0.219393  0.000000
  127.4865  172.4909  238.5493  370.9081  382.4981  436.8983
  607.3855  628.1229  670.6278
            0.000000  0.207846  0.000000
  121.1385  166.1689  231.5542  367.3449  383.7427  425.7046
  594.1990  618.1846  672.4504
            0.000000  0.196299  0.000000
  114.8550  159.5033  223.9856  363.9573  384.9878  415.0131
  582.2001  608.7202  674.1711
            0.000000  0.184752  0.000000
  108.6189  152.4862  215.7766  360.7591  386.2168  404.9264
  571.6149  599.8978  675.7994
            0.000000  0.173205  0.000000
  102.4087  145.1144  206.8692  357.7613  387.4140  395.5322
  562.6256  591.8636  677.3413
            0.000000  0.161658  0.000000
   96.2000  137.3890  197.2205  354.9716  386.8942  388.5648
  555.3584  584.7356  678.7995
            0.000000  0.150111  0.000000
   89.9667  129.3154  186.8083  352.3933  379.0470  389.6566
  549.8737  578.5970  680.1746
            0.000000  0.138564  0.000000
   83.6831  120.9026  175.6331  350.0255  371.9954  390.6785
  546.1590  573.4920  681.4651
            0.000000  0.127017  0.000000
   77.3248  112.1625  163.7170  347.8640  365.7199  391.6222
  544.1266  569.4235  682.6684
            0.000000  0.115470  0.000000
   70.8708  103.1092  151.0998  345.9027  360.1838  392.4814
  543.6167  566.3529  683.7811
            0.000000  0.103923  0.000000
   64.3040   93.7585  137.8344  344.1347  355.3419  393.2524
  544.4075  564.2030  684.7994
            0.000000  0.092376  0.000000
   57.6122   84.1283  123.9821  342.5543  351.1470  393.9333
  546.2298  562.8631  685.7197
            0.000000  0.080829  0.000000
   50.7890   74.2383  109.6084  341.1575  347.5542  394.5245
  548.7857  562.1970  686.5386
            0.000000  0.069282  0.000000
   43.8340   64.1110   94.7806  339.9422  344.5231  395.0274
  551.7672  562.0520  687.2532
            0.000000  0.057735  0.000000
   36.7526   53.7716   79.5664  338.9086  342.0183  395.4447
  554.8743  562.2695  687.8612
            0.000000  0.046188  0.000000
   29.5557   43.2486   64.0326  338.0581  340.0102  395.7798
  557.8305  562.6952  688.3606
            0.000000  0.034641  0.000000
   22.2597   32.5739   48.2448  337.3929  338.4740  396.0359
  560.3949  563.1885  688.7502
            0.000000  0.023094  0.000000
   14.8853   21.7823   32.2678  336.9156  337.3907  396.2163
  562.3715  563.6312  689.0291
            0.000000  0.011547  0.000000
    7.4565   10.9113   16.1652  336.6283  336.7462  396.3235
  563.6163  563.9339  689.1966
            0.000000  0.000000  0.000000
    0.0000    0.0000    0.0000  336.5323  336.5323  396.3590
  564.0411  564.0411  689.2525
            0.000000  0.000000  0.009128
    4.9723    4.9723    9.5605  336.4862  336.4862  396.3652
  564.0281  564.0281  689.2271
            0.000000  0.000000  0.018256
    9.9483    9.9483   19.1148  336.3478  336.3478  396.3844
  563.9884  563.9884  689.1500
            0.000000  0.000000  0.027384
   14.9313   14.9313   28.6564  336.1169  336.1169  396.4181
  563.9201  563.9201  689.0183
            0.000000  0.000000  0.036511
   19.9238   19.9238   38.1772  335.7934  335.7934  396.4689
  563.8203  563.8203  688.8272
            0.000000  0.000000  0.045639
   24.9296   24.9296   47.6717  335.3769  335.3769  396.5404
  563.6847  563.6847  688.5703
            0.000000  0.000000  0.054767
   29.9499   29.9499   57.1306  334.8669  334.8669  396.6365
  563.5082  563.5082  688.2398
            0.000000  0.000000  0.063895
   34.9859   34.9859   66.5443  334.2628  334.2628  396.7619
  563.2848  563.2848  687.8270
            0.000000  0.000000  0.073023
   40.0380   40.0380   75.9026  333.5638  333.5638  396.9214
  563.0081  563.0081  687.3223
            0.000000  0.000000  0.082151
   45.1053   45.1053   85.1938  332.7690  332.7690  397.1197
  562.6708  562.6708  686.7160
            0.000000  0.000000  0.091278
   50.1859   50.1859   94.4046  331.8773  331.8773  397.3612
  562.2659  562.2659  685.9986
            0.000000  0.000000  0.100406
   55.2782   55.2782  103.5236  330.8872  330.8872  397.6496
  561.7860  561.7860  685.1607
            0.000000  0.000000  0.109534
   60.3783   60.3783  112.5358  329.7972  329.7972  397.9876
  561.2238  561.2238  684.1945
            0.000000  0.000000  0.118662
   65.4818   65.4818  121.4265  328.6054  328.6054  398.3766
  560.5728  560.5728  683.0932
            0.000000  0.000000  0.127790
   70.5836   70.5836  130.1810  327.3100  327.3100  398.8165
  559.8269  559.8269  681.8517
            0.000000  0.000000  0.136918
   75.6782   75.6782  138.7849  325.9086  325.9086  399.3052
  558.9807  558.9807  680.4670
            0.000000  0.000000  0.146046
   80.7591   80.7591  147.2241  324.3991  324.3991  399.8386
  558.0301  558.0301  678.9384
            0.000000  0.000000  0.155173
   85.8194   85.8194  155.4848  322.7790  322.7790  400.4101
  556.9720  556.9720  677.2678
            0.000000  0.000000  0.164301
   90.8532   90.8532  163.5570  321.0455  321.0455  401.0111
  555.8044  555.8044  675.4590
            0.000000  0.000000  0.173429
   95.8531   95.8531  171.4298  319.1963  319.1963  401.6303
  554.5268  554.5268  673.5189
            0.000000  0.000000  0.182557
  100.8124  100.8124  179.0953  317.2290  317.2290  402.2540
  553.1405  553.1405  671.4570
            0.000000  0.000000  0.191685
  105.7245  105.7245  186.5476  315.1411  315.1411  402.8662
  551.6477  551.6477  669.2849
            0.000000  0.000000  0.200813
  110.5833  110.5833  193.7833  312.9308  312.9308  403.4492
  550.0528  550.0528  667.0167
            0.000000  0.000000  0.209940
  115.3827  115.3827  200.8006  310.5965  310.5965  403.9833
  548.3614  548.3614  664.6684
            0.000000  0.000000  0.219068
  120.1189  120.1189  207.6022  308.1366  308.1366  404.4478
  546.5804  546.5804  662.2573
            0.000000  0.000000  0.228196
  124.7874  124.7874  214.1908  305.5506  305.5506  404.8213
  544.7183  544.7183  659.8024
            0.000000  0.000000  0.237324
  129.3848  129.3848  220.5715  302.8386  302.8386  405.0825
  542.7853  542.7853  657.3234
            0.000000  0.000000  0.246452
  133.9091  133.9091  226.7510  300.0013  300.0013  405.2103
  540.7923  540.7923  654.8405
            0.000000  0.000000  0.255580
  138.3590  138.3590  232.7372  297.0403  297.0403  405.1853
  538.7517  538.7517  652.3740
            0.000000  0.000000  0.264707
  142.7339  142.7339  238.5379  293.9586  293.9586  404.9898
  536.6769  536.6769  649.9438
            0.000000  0.000000  0.273835
  147.0357  147.0357  244.1630  290.7589  290.7589  404.6089
  534.5816  534.5816  647.5684
            0.000000  0.000000  0.282963
  151.2658  151.2658  249.6200  287.4463  287.4463  404.0306
  532.4806  532.4806  645.2657
            0.000000  0.000000  0.292091
  155.4272  155.4272  254.9164  284.0263  284.0263  403.2469
  530.3892  530.3892  643.0519
            0.000000  0.000000  0.301219
  159.5237  159.5237  260.0583  280.5058  280.5058  402.2540
  528.3227  528.3227  640.9413
            0.000000  0.000000  0.310347
  163.5600  163.5600  265.0496  276.8924  276.8924  401.0528
  526.2969  526.2969  638.9462
            0.000000  0.000000  0.319475
  167.5413  167.5413  269.8922  273.1951  273.1951  399.6494
  524.3271  524.3271  637.0766
            0.000000  0.000000  0.328602
  171.4729  171.4729  269.4242  269.4242  274.5839  398.0556
  522.4289  522.4289  635.3407
            0.000000  0.000000  0.337730
  175.3617  175.3617  265.5895  265.5895  279.1205  396.2885
  520.6167  520.6167  633.7437
            0.000000  0.000000  0.346858
  179.2130  179.2130  261.7035  261.7035  283.4916  394.3722
  518.9049  518.9049  632.2897
            0.000000  0.000000  0.355986
  183.0322  183.0322  257.7790  257.7790  287.6814  392.3380
  517.3072  517.3072  630.9803
            0.000000  0.000000  0.365114
  186.8232  186.8232  253.8303  253.8303  291.6674  390.2245
  515.8363  515.8363  629.8156
            0.000000  0.000000  0.374242
  190.5880  190.5880  249.8735  249.8735  295.4188  388.0794
  514.5038  514.5038  628.7947
            0.000000  0.000000  0.383369
  194.3242  194.3242  245.9282  245.9282  298.8948  385.9597
  513.3206  513.3206  627.9152
            0.000000  0.000000  0.392497
  198.0237  198.0237  242.0189  242.0189  302.0450  383.9321
  512.2959  512.2959  627.1743
            0.000000  0.000000  0.401625
  201.6634  201.6634  238.1831  238.1831  304.8067  382.0735
  511.4381  511.4381  626.5689
            0.000000  0.000000  0.410753
  205.1914  205.1914  234.4856  234.4856  307.1086  380.4683
  510.7539  510.7539  626.0957
            0.000000  0.000000  0.419881
  208.4846  208.4846  231.0598  231.0598  308.8750  379.2030
  510.2488  510.2488  625.7520
            0.000000  0.000000  0.429009
  211.2336  211.2336  228.2230  228.2230  310.0347  378.3566
  509.9270  509.9270  625.5354
            0.000000  0.000000  0.438137
  212.7190  212.7190  226.6997  226.6997  310.5338  377.9887
  509.7909  509.7909  625.4444
            0.000000  0.000000  0.447264
  212.1299  212.1299  227.3030  227.3030  310.3469  378.1267
  509.8417  509.8417  625.4783
            0.000000  0.000000  0.456392
  209.8386  209.8386  229.6598  229.6598  309.4836  378.7603
  510.0790  510.0790  625.6374
            0.000000  0.011547  0.456392
  210.2734  210.5402  229.7629  229.8253  309.5442  378.8089
  510.0133  510.0545  625.6688
            0.000000  0.023094  0.456392
  211.5504  212.5720  230.0874  230.3643  309.7307  378.9552
  509.8390  510.0075  625.7617
            0.000000  0.034641  0.456392
  213.5811  215.6836  230.6856  231.4371  310.0587  379.2008
  509.6226  510.0165  625.9127
            0.000000  0.046188  0.456392
  216.1964  219.3517  231.6705  233.4226  310.5583  379.5483
  509.4711  510.2067  626.1161
            0.000000  0.057735  0.456392
  219.1112  222.7511  233.2566  236.9517  311.2811  380.0008
  509.5249  510.7431  626.3641
            0.000000  0.069282  0.456392
  221.9197  225.2812  235.7704  242.3814  312.3140  380.5620
  509.9495  511.8197  626.6475
            0.000000  0.080829  0.456392
  224.2561  227.0685  239.4951  249.2824  313.7998  381.2359
  510.9243  513.6457  626.9554
            0.000000  0.092376  0.456392
  226.0431  228.4720  244.4267  256.9130  315.9682  382.0260
  512.6304  516.4309  627.2760
            0.000000  0.103923  0.456392
  227.4366  229.7176  250.3324  264.5621  319.1636  382.9355
  515.2384  520.3687  627.5967
            0.000000  0.115470  0.456392
  228.6080  230.9160  256.9676  271.5528  323.8265  383.9670
  518.8955  525.6215  627.9046
            0.000000  0.127017  0.456392
  229.6682  232.1218  264.1513  277.3221  330.3541  385.1220
  523.7156  532.3078  628.1866
            0.000000  0.138564  0.456392
  230.6795  233.3633  271.7521  281.6132  338.8752  386.4014
  529.7713  540.4935  628.4298
            0.000000  0.150111  0.456392
  231.6752  234.6561  279.6658  284.5340  349.1724  387.8061
  537.0893  550.1875  628.6221
            0.000000  0.161658  0.456392
  232.6733  236.0097  286.3877  287.8009  360.8398  389.3401
  545.6486  561.3413  628.7529
            0.000000  0.173205  0.456392
  233.6826  237.4298  287.4826  296.0728  373.4526  391.0249
  555.3817  573.8509  628.8156
            0.000000  0.184752  0.456392
  234.7068  238.9199  288.0548  304.4022  386.4694  393.0737
  566.1779  587.5561  628.8135
            0.000000  0.196299  0.456392
  235.7464  240.4823  288.2671  312.7168  393.9345  401.1914
  577.8888  602.2173  628.7908
            0.000000  0.207846  0.456392
  236.7998  242.1178  288.2280  320.9533  396.1294  414.9061
  590.3352  617.2437  629.1124
            0.000000  0.219393  0.456392
  237.8642  243.8262  288.0096  329.0605  398.1828  428.9442
  603.3151  626.2181  635.9478
            0.000000  0.230940  0.456392
  238.9355  245.6063  287.6603  337.0002  400.2709  443.0187
  616.6120  626.5413  651.6354
            0.000000  0.242487  0.456392
  240.0094  247.4563  287.2132  344.7475  402.4090  457.0310
  625.9801  630.0044  668.1429
            0.000000  0.254034  0.456392
  241.0810  249.3731  286.6913  352.2896  404.5955  470.9215
  625.1296  643.2736  684.6134
            0.000000  0.265581  0.456392
  242.1452  251.3533  286.1110  359.6236  406.8251  484.6509
  624.0508  656.2114  700.7346
            0.000000  0.277128  0.456392
  243.1971  253.3926  285.4846  366.7539  409.0924  498.1941
  622.7495  668.6269  716.2666
            0.000000  0.288675  0.456392
  244.2317  255.4864  284.8218  373.6889  411.3924  511.5353
  621.2191  680.3515  731.0040
            0.000000  0.300222  0.456392
  245.2442  257.6296  284.1305  380.4385  413.7215  524.6651
  619.4485  691.2430  744.7718
            0.000000  0.311769  0.456392
  246.2304  259.8165  283.4182  387.0110  416.0776  537.5776
  617.4253  701.1883  757.4247
            0.000000  0.323316  0.456392
  247.1860  262.0411  282.6922  393.4107  418.4600  550.2685
  615.1372  710.1049  768.8480
            0.000000  0.334863  0.456392
  248.1074  264.2957  281.9606  399.6369  420.8701  562.7335
  612.5722  717.9412  778.9578
            0.000000  0.346410  0.456392
  248.9915  266.5707  281.2340  405.6818  423.3109  574.9671
  609.7201  724.6762  787.6999
            0.000000  0.357957  0.456392
  249.8354  268.8503  280.5281  411.5310  425.7873  586.9616
  606.5723  730.3181  795.0494
            0.000000  0.369504  0.456392
  250.6370  271.1038  279.8739  417.1631  428.3057  598.7059
  603.1245  734.9018  801.0083
            0.000000  0.381051  0.456392
  251.3944  273.2513  279.3506  422.5514  430.8744  599.3675
  610.1941  738.4863  805.6035
            0.000000  0.392598  0.456392
  252.1063  275.0308  279.2188  427.6649  433.5031  595.3109
  621.4035  741.1508  808.8843
            0.000000  0.404145  0.456392
  252.7718  275.8403  280.0776  432.4708  436.2026  590.9560
  632.3226  742.9907  810.9190
            0.000000  0.415692  0.456392
  253.3904  275.7127  281.8897  436.9373  438.9851  586.3132
  642.9344  744.1126  811.7918
            0.000000  0.427239  0.456392
  253.9620  275.2473  284.0488  441.0355  441.8632  581.3980
  653.2226  744.6301  811.5989
            0.000000  0.438786  0.456392
  254.4868  274.6902  286.2994  444.7429  444.8498  576.2316
  663.1723  744.6582  810.4450
            0.000000  0.450333  0.456392
  254.9650  274.1166  288.5541  447.9571  448.0449  570.8413
  672.7707  744.3094  808.4393
            0.000000  0.461880  0.456392
  255.3975  273.5556  290.7678  450.9366  451.1960  565.2613
  682.0085  743.6891  805.6919
            0.000000  0.473427  0.456392
  255.7849  273.0223  292.9067  453.4236  454.5741  559.5333
  690.8809  742.8929  802.3099
            0.000000  0.484974  0.456392
  256.1281  272.5262  294.9395  455.5215  458.0940  553.7074
  699.3878  742.0040  798.3944
            0.000000  0.496521  0.456392
  256.4281  272.0741  296.8350  457.2549  461.7490  547.8449
  707.5347  741.0921  794.0373
            0.000000  0.508068  0.456392
  256.6858  271.6716  298.5617  458.6548  465.5183  542.0220
  715.3327  740.2138  789.3193
            0.000000  0.519615  0.456392
  256.9021  271.3233  300.0883  459.7566  469.3566  536.3376
  722.7979  739.4130  784.3080
            0.000000  0.531162  0.456392
  257.0778  271.0329  301.3852  460.5964  473.1775  530.9281
  729.9501  738.7223  779.0572
            0.000000  0.542709  0.456392
  257.2137  270.8037  302.4254  461.2086  476.8229  525.9964
  736.8110  738.1651  773.6081
            0.000000  0.554256  0.456392
  257.3102  270.6381  303.1860  461.6227  480.0137  521.8586
  737.7574  743.3972  767.9937
            0.000000  0.565803  0.456392
  257.3680  270.5380  303.6496  461.8616  482.3033  518.9893
  737.5091  749.6861  762.2732
            0.000000  0.577350  0.456392
  257.3873  270.5046  303.8054  461.9396  483.1563  517.9411
  737.4258  754.4850  757.6613
