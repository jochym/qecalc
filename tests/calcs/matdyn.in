&INPUT
    asr = 'crystal',
    amass(1) = 24.305,
    amass(2) = 11.000,
    flfrc = 'mgb2.fc',
/
151
 0.00000000  0.57735027  0.00000000  0.00000000
 0.00000000  0.56580326  0.00000000  0.01154701
 0.00000000  0.55425626  0.00000000  0.02309401
 0.00000000  0.54270925  0.00000000  0.03464102
 0.00000000  0.53116225  0.00000000  0.04618802
 0.00000000  0.51961524  0.00000000  0.05773503
 0.00000000  0.50806824  0.00000000  0.06928203
 0.00000000  0.49652123  0.00000000  0.08082904
 0.00000000  0.48497423  0.00000000  0.09237604
 0.00000000  0.47342722  0.00000000  0.10392305
 0.00000000  0.46188022  0.00000000  0.11547005
 0.00000000  0.45033321  0.00000000  0.12701706
 0.00000000  0.43878620  0.00000000  0.13856406
 0.00000000  0.42723920  0.00000000  0.15011107
 0.00000000  0.41569219  0.00000000  0.16165808
 0.00000000  0.40414519  0.00000000  0.17320508
 0.00000000  0.39259818  0.00000000  0.18475209
 0.00000000  0.38105118  0.00000000  0.19629909
 0.00000000  0.36950417  0.00000000  0.20784610
 0.00000000  0.35795717  0.00000000  0.21939310
 0.00000000  0.34641016  0.00000000  0.23094011
 0.00000000  0.33486316  0.00000000  0.24248711
 0.00000000  0.32331615  0.00000000  0.25403412
 0.00000000  0.31176915  0.00000000  0.26558112
 0.00000000  0.30022214  0.00000000  0.27712813
 0.00000000  0.28867513  0.00000000  0.28867513
 0.00000000  0.27712813  0.00000000  0.30022214
 0.00000000  0.26558112  0.00000000  0.31176915
 0.00000000  0.25403412  0.00000000  0.32331615
 0.00000000  0.24248711  0.00000000  0.33486316
 0.00000000  0.23094011  0.00000000  0.34641016
 0.00000000  0.21939310  0.00000000  0.35795717
 0.00000000  0.20784610  0.00000000  0.36950417
 0.00000000  0.19629909  0.00000000  0.38105118
 0.00000000  0.18475209  0.00000000  0.39259818
 0.00000000  0.17320508  0.00000000  0.40414519
 0.00000000  0.16165808  0.00000000  0.41569219
 0.00000000  0.15011107  0.00000000  0.42723920
 0.00000000  0.13856406  0.00000000  0.43878620
 0.00000000  0.12701706  0.00000000  0.45033321
 0.00000000  0.11547005  0.00000000  0.46188022
 0.00000000  0.10392305  0.00000000  0.47342722
 0.00000000  0.09237604  0.00000000  0.48497423
 0.00000000  0.08082904  0.00000000  0.49652123
 0.00000000  0.06928203  0.00000000  0.50806824
 0.00000000  0.05773503  0.00000000  0.51961524
 0.00000000  0.04618802  0.00000000  0.53116225
 0.00000000  0.03464102  0.00000000  0.54270925
 0.00000000  0.02309401  0.00000000  0.55425626
 0.00000000  0.01154701  0.00000000  0.56580326
 0.00000000  0.00000000  0.00000000  0.57735027
 0.00000000  0.00000000  0.00912784  0.58647811
 0.00000000  0.00000000  0.01825569  0.59560596
 0.00000000  0.00000000  0.02738353  0.60473380
 0.00000000  0.00000000  0.03651138  0.61386165
 0.00000000  0.00000000  0.04563922  0.62298949
 0.00000000  0.00000000  0.05476706  0.63211733
 0.00000000  0.00000000  0.06389491  0.64124518
 0.00000000  0.00000000  0.07302275  0.65037302
 0.00000000  0.00000000  0.08215060  0.65950087
 0.00000000  0.00000000  0.09127844  0.66862871
 0.00000000  0.00000000  0.10040629  0.67775655
 0.00000000  0.00000000  0.10953413  0.68688440
 0.00000000  0.00000000  0.11866197  0.69601224
 0.00000000  0.00000000  0.12778982  0.70514009
 0.00000000  0.00000000  0.13691766  0.71426793
 0.00000000  0.00000000  0.14604551  0.72339578
 0.00000000  0.00000000  0.15517335  0.73252362
 0.00000000  0.00000000  0.16430119  0.74165146
 0.00000000  0.00000000  0.17342904  0.75077931
 0.00000000  0.00000000  0.18255688  0.75990715
 0.00000000  0.00000000  0.19168473  0.76903500
 0.00000000  0.00000000  0.20081257  0.77816284
 0.00000000  0.00000000  0.20994041  0.78729068
 0.00000000  0.00000000  0.21906826  0.79641853
 0.00000000  0.00000000  0.22819610  0.80554637
 0.00000000  0.00000000  0.23732395  0.81467422
 0.00000000  0.00000000  0.24645179  0.82380206
 0.00000000  0.00000000  0.25557964  0.83292990
 0.00000000  0.00000000  0.26470748  0.84205775
 0.00000000  0.00000000  0.27383532  0.85118559
 0.00000000  0.00000000  0.28296317  0.86031344
 0.00000000  0.00000000  0.29209101  0.86944128
 0.00000000  0.00000000  0.30121886  0.87856913
 0.00000000  0.00000000  0.31034670  0.88769697
 0.00000000  0.00000000  0.31947454  0.89682481
 0.00000000  0.00000000  0.32860239  0.90595266
 0.00000000  0.00000000  0.33773023  0.91508050
 0.00000000  0.00000000  0.34685808  0.92420835
 0.00000000  0.00000000  0.35598592  0.93333619
 0.00000000  0.00000000  0.36511376  0.94246403
 0.00000000  0.00000000  0.37424161  0.95159188
 0.00000000  0.00000000  0.38336945  0.96071972
 0.00000000  0.00000000  0.39249730  0.96984757
 0.00000000  0.00000000  0.40162514  0.97897541
 0.00000000  0.00000000  0.41075299  0.98810325
 0.00000000  0.00000000  0.41988083  0.99723110
 0.00000000  0.00000000  0.42900867  1.00635894
 0.00000000  0.00000000  0.43813652  1.01548679
 0.00000000  0.00000000  0.44726436  1.02461463
 0.00000000  0.00000000  0.45639221  1.03374248
 0.00000000  0.01154701  0.45639221  1.04528948
 0.00000000  0.02309401  0.45639221  1.05683649
 0.00000000  0.03464102  0.45639221  1.06838349
 0.00000000  0.04618802  0.45639221  1.07993050
 0.00000000  0.05773503  0.45639221  1.09147750
 0.00000000  0.06928203  0.45639221  1.10302451
 0.00000000  0.08082904  0.45639221  1.11457151
 0.00000000  0.09237604  0.45639221  1.12611852
 0.00000000  0.10392305  0.45639221  1.13766552
 0.00000000  0.11547005  0.45639221  1.14921253
 0.00000000  0.12701706  0.45639221  1.16075953
 0.00000000  0.13856406  0.45639221  1.17230654
 0.00000000  0.15011107  0.45639221  1.18385355
 0.00000000  0.16165808  0.45639221  1.19540055
 0.00000000  0.17320508  0.45639221  1.20694756
 0.00000000  0.18475209  0.45639221  1.21849456
 0.00000000  0.19629909  0.45639221  1.23004157
 0.00000000  0.20784610  0.45639221  1.24158857
 0.00000000  0.21939310  0.45639221  1.25313558
 0.00000000  0.23094011  0.45639221  1.26468258
 0.00000000  0.24248711  0.45639221  1.27622959
 0.00000000  0.25403412  0.45639221  1.28777659
 0.00000000  0.26558112  0.45639221  1.29932360
 0.00000000  0.27712813  0.45639221  1.31087060
 0.00000000  0.28867513  0.45639221  1.32241761
 0.00000000  0.30022214  0.45639221  1.33396462
 0.00000000  0.31176915  0.45639221  1.34551162
 0.00000000  0.32331615  0.45639221  1.35705863
 0.00000000  0.33486316  0.45639221  1.36860563
 0.00000000  0.34641016  0.45639221  1.38015264
 0.00000000  0.35795717  0.45639221  1.39169964
 0.00000000  0.36950417  0.45639221  1.40324665
 0.00000000  0.38105118  0.45639221  1.41479365
 0.00000000  0.39259818  0.45639221  1.42634066
 0.00000000  0.40414519  0.45639221  1.43788766
 0.00000000  0.41569219  0.45639221  1.44943467
 0.00000000  0.42723920  0.45639221  1.46098167
 0.00000000  0.43878620  0.45639221  1.47252868
 0.00000000  0.45033321  0.45639221  1.48407569
 0.00000000  0.46188022  0.45639221  1.49562269
 0.00000000  0.47342722  0.45639221  1.50716970
 0.00000000  0.48497423  0.45639221  1.51871670
 0.00000000  0.49652123  0.45639221  1.53026371
 0.00000000  0.50806824  0.45639221  1.54181071
 0.00000000  0.51961524  0.45639221  1.55335772
 0.00000000  0.53116225  0.45639221  1.56490472
 0.00000000  0.54270925  0.45639221  1.57645173
 0.00000000  0.55425626  0.45639221  1.58799873
 0.00000000  0.56580326  0.45639221  1.59954574
 0.00000000  0.57735027  0.45639221  1.61109274
