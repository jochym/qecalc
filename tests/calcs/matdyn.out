Norm of the difference between old and new effective charges:    0.00000000000000000000
Norm of the difference between old and new force-constants:   0.00011075535746680777
     Message from routine matdyn:
     Z* not found in file mgb2666.fc, TO-LO splitting at q=0 will be absent!
     A direction for q was not specified:TO-LO splitting will be absent
